libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -1,340 +1,187 @@
+        <div className="z-10">
+          <AccountsActionsDropdown
+            isReadOnly={isReadOnly}
+            assetId={asset.id}
+            assetContractAddress={
+              asset.source?.__typename === 'ERC20'
+                ? asset.source.contractAddress
+                : undefined
+            }
+            onClickDeposit={() => {
+              actions.onClickDeposit?.(asset.id);
+            }}
+            onClickWithdraw={() => {
+              actions.onClickWithdraw?.(asset.id);
+            }}
+            onClickTransfer={() => {
+              actions.onClickTransfer?.(asset.id);
+            }}
+          />
+        </div>
+      </div>
+      {data?.breakdown?.map((data) => (
+        <BreakdownItem data={data} />
+      ))}
+      <div className="flex items-center mt-3">
+        <div className="text-base font-alpha">{t('Total')}</div>
+        <div className="text-right leading-tight font-alpha grow text-base">
+          {addDecimalsFormatNumberQuantum(
+            data?.total || '0',
+            asset.decimals,
+            asset.quantum
+          )}
+        </div>
+      </div>
+      {expanded ? (
+        <div className="grid gap-1 grid-cols-4 mt-3">
+          <Button
+            onClick={() => actions.onClickDeposit?.(asset.id)}
+            label={t('Deposit')}
+            icon={VegaIconNames.DEPOSIT}
+          />
+          <Button
+            onClick={() => actions.onClickSwap?.(asset.id)}
+            label={t('Swap')}
+            icon={VegaIconNames.SWAP}
+          />
+          <Button
+            onClick={() => actions.onClickTransfer?.(asset.id)}
+            label={t('Transfer')}
+            icon={VegaIconNames.TRANSFER}
+          />
+          <Button
+            onClick={() => actions.onClickWithdraw?.(asset.id)}
+            label={t('Withdraw')}
+            icon={VegaIconNames.WITHDRAW}
+          />
+        </div>
+      ) : (
+        <button
+          className="absolute inset-0"
+          onClick={() => setExpanded(true)}
+        ></button>
+      )}
+    </div>
+  );
+};
+export const AccountTable = ({
+  rowData,
+  pinnedAssets,
+  ...props
+}: AccountTableProps) => {
   return (
-    <AgGrid
-      {...props}
-      getRowId={({ data }: { data: AccountFields }) => data.asset.id}
-      tooltipShowDelay={500}
-      rowData={data}
-      defaultColDef={defaultColDef}
-      columnDefs={colDefs}
-      overlayNoRowsTemplate={
-        // account for the pinned asset is filtered out to prevent duplicate
-        // data in the pinned row and the main table rows. AgGrid will not
-        // consider the pinned row when determining whether to show the no
-        // rows template or not so we need to override it
-        rowData?.length ? '<span />' : props.overlayNoRowsTemplate
-      }
-      getRowHeight={getPinnedAssetRowHeight}
-      pinnedTopRowData={pinnedRow ? [pinnedRow] : undefined}
-    />
+    rowData &&
+    (pinnedAssets && pinnedAssets.length
+      ? [
```

Comment:
I believe this is just for sorting purposes?

This would show intention more explicitly
```
 ? sortBy(rowData, (asset) => {
      if (pinnedAssets.includes(asset.id)) return -1
      return 1
    })
    : rowData
```